src/BUILD

```diff
@@ -245,6 +245,15 @@ cc_binary(
     linkshared = 1,
 )
 
+cc_binary(
```

Comment:
Maybe we could make use of `createLibraryMock<>` instead of making new custom node binary?

Reply:
We can use LibraryMock instead, however I assume that we would only validate `pipelineDefinition` instead of creating the pipeline and executing. If that is fine then we can get rid of that CN and replace the tests with Mock.